runtime-decompiler/src/main/java/org/jrd/backend/data/Cli.java

```diff
@@ -614,62 +616,42 @@ private void listJvms(List<String> args) {
         }
     }
 
-    private void printVersion() throws IOException {
-        Enumeration<URL> resources = getClass().getClassLoader().getResources("META-INF/MANIFEST.MF");
+    protected static Optional<Attributes> getJrdAttributes() throws IOException {
+        Enumeration<URL> resources = Cli.class.getClassLoader().getResources("META-INF/MANIFEST.MF");
+
         while (resources.hasMoreElements()) {
             Manifest manifest = new Manifest(resources.nextElement().openStream());
-            Attributes w = manifest.getAttributes("runtime-decompiler");
-            if (w!=null) {
-                if ("java-runtime-decompiler".equals(w.getValue("groupId")) &&
-                        "runtime-decompiler".equals(w.getValue("artifactId"))) {
-                    System.out.println((w.getValue("groupId")) + " JRD " + w.getValue("version") + " - " + w.getValue("timestamp"));
-                    return;
+            Attributes attributes = manifest.getAttributes("runtime-decompiler");
+
+            if (attributes != null) {
+                if ("java-runtime-decompiler".equals(attributes.getValue("groupId")) &&
+                        "runtime-decompiler".equals(attributes.getValue("artifactId"))) {
+                    return Optional.of(attributes);
```

Comment:
Can you pleae instead of attributes class,retun your own object? And Extract  it to separate method eg getBuildTimeManifestInformation()?

Later just call this method, get this class, and access  its getGoup, getDate, getVersion...